### PR TITLE
S3t2 create form page

### PR DESCRIPTION
**Pull request by Tibereon**

### CODE REVIEW

File: templates/createEventsPage.phtml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
+    <title>Registered Applicants</title>
```
Comment (MotNosniktaw) [inline templates/createEventsPage.phtml]: This should be 'Academy Portal | Events'

File: templates/createEventsPage.phtml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="../public/css/createEventsPage.css">
```
Comment (MotNosniktaw) [inline templates/createEventsPage.phtml]: fix paths for production version - should be ./css/......

Comment (MotNosniktaw) [reply templates/createEventsPage.phtml]: does this stylesheet exist?

File: templates/createEventsPage.phtml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+        <label for='EventCategory'>Event category</label>
+        <select name='eventCategory' id='eventCategory' class='createEvents form-control'>
+            <option value='0'>Select and event category</option>
+            <?php echo \Portal\ViewHelpers\EventCategoryViewHelper::eventCategoryDropdown($data); ?>
+        </select>
+
+        <label for="eventName">Event Name</label>
+        <input id="eventName" name="eventName" class="createEvents form-control" type="text" placeholder="Event Name"
+               data-required='true' data-max='255'>
```
Comment (MotNosniktaw) [inline templates/createEventsPage.phtml]: are there no generic/shared style classes with hiring partner

File: templates/createEventsPage.phtml (hunk 1 above)
Comment (Tibereon) [reply templates/createEventsPage.phtml]: on it

File: templates/createEventsPage.phtml (hunk 2 above)
Comment (Tibereon) [reply templates/createEventsPage.phtml]: on it

File: templates/createEventsPage.phtml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <link rel="stylesheet" type="text/css" href="../public/css/genericStyles.css">
+</head>
+<body>
+<div class="container">
+    <h1>Events</h1>
+    <button type="button" class="btn btn-success">go back</button>
+    <h2>Add new Event</h2>
+    <form id='CreateEventsForm'>
+
+        <label for='EventCategory'>Event category</label>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: Label doesn't match id

File: templates/createEventsPage.phtml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        <select name='eventCategory' id='eventCategory' class='createEvents form-control'>
+            <option value='0'>Select and event category</option>
+            <?php echo \Portal\ViewHelpers\EventCategoryViewHelper::eventCategoryDropdown($data); ?>
+        </select>
+
+        <label for="eventName">Event Name</label>
+        <input id="eventName" name="eventName" class="createEvents form-control" type="text" placeholder="Event Name"
+               data-required='true' data-max='255'>
+
+        <label for='eventDate'>date</label>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: label doesnt match id

File: templates/createEventsPage.phtml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        <input id="EventDate" name="name" class="createEvents form-control" type="date" data-required='true'
+               data-max='255'>
+
+        <label for="eventLocation">location</label>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: label doesn't match id

File: templates/createEventsPage.phtml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        <input id="location" name="location" class="createEvents form-control" type="text" placeholder="Location"
+               data-required='true' data-max='600'>
+
+        <label for='eventStartTime'>Event start time</label>
+        <input id='eventStartTime' name='EventStartTime' class="createEvents form-control" type="time" data-max='20'>
+
+        <label for='eventEndTime'>Event end time</label>
+        <input id='eventEndTime' name='EventEndTime' class="createEvents form-control" type="time" data-max='20'>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: Times don't have data required but shouldn't be optional

File: templates/createEventsPage.phtml [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+        <label for='eventDate'>date</label>
+        <input id="EventDate" name="name" class="createEvents form-control" type="date" data-required='true'
+               data-max='255'>
+
+        <label for="eventLocation">location</label>
+        <input id="location" name="location" class="createEvents form-control" type="text" placeholder="Location"
+               data-required='true' data-max='600'>
+
+        <label for='eventStartTime'>Event start time</label>
+        <input id='eventStartTime' name='EventStartTime' class="createEvents form-control" type="time" data-max='20'>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: Times don't have data required but shouldn't be optiona

File: templates/createEventsPage.phtml (hunk 7 above)
Comment (Tibereon) [reply templates/createEventsPage.phtml]: On It

File: templates/createEventsPage.phtml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        <label for='eventEndTime'>Event end time</label>
+        <input id='eventEndTime' name='eventEndTime' class="createEvents form-control" type="time" data-required='true'
+               data-max='20'>
+
+        <label for='eventNotes'>Notes</label>
+        <input id='eventNotes' name='eventNotes' class="createEvents form-control" type="text" placeholder="Notes"
+               data-max='255'>
+        <input id="submitEvent" type="submit" class="btn btn-success"/>
+    </form>
+</div>
```
Comment (mallowmew) [inline templates/createEventsPage.phtml]: Where does the list of events go? Thought we were replicating the Hiring Partners page

File: templates/createEventsPage.phtml [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="./css/genericStyles.css">
+</head>
+<body>
+<div class="container">
+    <h1>Events</h1>
+    <button type="button" class="btn btn-success">go back</button>
+    <h2>Add new Event</h2>
+    <form id='createEventsForm'>
```
Comment (mallowmew) [inline templates/createEventsPage.phtml]: ids and class names should be kebab-case

**Comment by FelixLetheren** [comment]
HTML should be kebab-vase not camelCase

File: templates/createEventsPage.phtml (hunk 9 above)
Comment (Tibereon) [reply templates/createEventsPage.phtml]: On it

File: templates/createEventsPage.phtml [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+        <label for="event-name">Event Name</label>
+        <input id="event-name" name="event-name" class="create-events form-control" type="text" placeholder="Event Name"
+               data-required='true' data-max='255'>
+
+        <label for='event-date'>date</label>
+        <input id="event-date" name="event-date" class="create-events form-control" type="date" data-required='true'
+               data-max='255'>
+
+        <label for="event-location">location</label>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: lower case when rest of labels are upper case.

File: templates/createEventsPage.phtml [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+        <select name='event-category' id='event-category' class='create-events form-control' data-required='true'>
+            <option value='0'>Select and event category</option>
+            <?php echo \Portal\ViewHelpers\EventCategoryViewHelper::eventCategoryDropdown($data); ?>
+        </select>
+
+        <label for="event-name">Event Name</label>
+        <input id="event-name" name="event-name" class="create-events form-control" type="text" placeholder="Event Name"
+               data-required='true' data-max='255'>
+
+        <label for='event-date'>date</label>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: lower case when rest of labels are upper case.

File: templates/createEventsPage.phtml [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
+    <title>Academy Events</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="./css/genericStyles.css">
+</head>
+<body>
+<div class="container">
+    <h1>Events</h1>
+    <button type="button" class="btn btn-success">go back</button>
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: Fot consistancy should be "Go back to the Admin Page"

File: templates/createEventsPage.phtml [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+    <h2>Add new Event</h2>
+    <form id='createEventsForm'>
+
```
Comment (FelixLetheren) [inline templates/createEventsPage.phtml]: Extra newline

Comment (Tibereon) [reply templates/createEventsPage.phtml]: ON IT

**Pull request merged**